boards/riscv/titanium_ti60_f225/doc/index.rst

```diff
@@ -0,0 +1,56 @@
+.. _titanium_ti60_f225:
+
+Efinix Titanium Ti60 F225
+#########################
+
+Overview
+********
+
+The Efinix Titanium Ti60 F225 development kit contains a Ti60 FPGA, which is fabricated on a 16nm process and deliver
+high performance with the lowest possible power on a small physical size. In addition, Efinix offers Sapphire SoC IP,
+which is a user-configurable RISC-V SoC based on the VexRiscv core with configurable feature set and extension.
```

Comment:
I don't see which Sapphire SoC revision (commit SHA?) this Zephyr platform assumes in this doc and in the linked Sapphire-Zephyr repo. Could you add it somewhere in the documentation? Or is it somewhere in the Sapphire-Zephyr repo, and I've just missed it?

Reply:
> Sorry, I don't quite understand "SoC revision (commit SHA?)"? Could you clarify, please?

Since you're using VexRiscv in your SoC, I assumed that your SoC is a soft one. Now, because soft SoC can be changed you risk making this Zephyr port not work. For some soft SoCs in Zephyr, e.g. OpenTitan, we have the SoC [revision specified in the board docs](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/riscv/opentitan_earlgrey/doc/index.rst#building-and-flashing). And by SoC revision I mean the commit SHA from the repository that hosts the SoC.

Another approach is taken by e.g. LiteX VexRiscv platform, which links to an external repository which always generates the same version of the soft SoC: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/riscv/litex_vexriscv/doc/index.rst#litex-vexriscv.

> Efinix Efinity IP Manager (software tool) generates the Sapphire SoC. Do you mean you mean the version of the software tool?

What I meant is that I want to have a description somewhere on how to generate the exact soft SoC that is Zephyr port assumes, immune to future changes. If giving the version of this tool guarantees that, then that's fine by me.

Reply:
Thanks!